### PR TITLE
github: initial CODEOWNERS setup for CI configuration

### PR DESCRIPTION
**Pull request by mback2k**
As discussed on IRC and the wiki: https://github.com/curl/curl/wiki/State-of-curl-CI

### CODE REVIEW

**Review by bagder — APPROVED** [review]

**Comment by mback2k** [comment]
@cmeister2 can I also get your approval being a CODEOWNER as proposed with this PR?

**Review by cmeister2 — APPROVED** [review]
LGTM.

**Pull request closed without merging**
